packages/libretro/gme/package.mk

```diff
@@ -24,7 +24,8 @@ PKG_REV="1"
 PKG_ARCH="any"
 PKG_LICENSE="GPLv3"
 PKG_SITE="https://github.com/libretro/libretro-gme"
-PKG_URL="https://github.com/libretro/libretro-gme/archive/$PKG_VERSION.tar.gz"
+PKG_SOURCE_DIR="libretro-gme-$PKG_VERSION"
```

Comment:
PKG_SOURCE_DIR? Is this needed? The other packages don't have it.

Reply:
Indeed, this is a leftover from the targz which is not needed anymore